test/BaselineOutput/Common/BinaryLoaderSaver/OldKeyTypeCodecTest.txt

```diff
@@ -0,0 +1,9 @@
+#@ TextLoader{
+#@   sep=tab
+#@   col=A:U4[3]:0
+#@   col=B:U4[6]:1
+#@   col=C:U1[11]:2
+#@   col=D:U2[65535]:3
+#@   col=E:U4[4294967295]:4
+#@   col=F:U8[18446744073709551615]:5
```

Comment:
>18446744073709551615 [](start = 14, length = 20)

Not a big deal at all, and probably this is the right thing, but when I read this I kind of wonder if rendering this as `U8[]` would be a better choice. :) Though it's probably best to be explicit, even if it is a bit verbose.

Reply:
I agree that it would look better. However, if conceptually we decide to require a defined Count for KeyType, for consistency I would argue in favor of keeping it in the string representation of the KeyType.

---
In reply to: [250352178](https://github.com/dotnet/machinelearning/pull/2146#discussion_r250352178) [](ancestors = 250352178)